server/lib/nl/detection/place.py

```diff
@@ -159,15 +168,24 @@ def detect_from_names(place_names: List[str],
     main_place = resolved_places[0]
     parent_places = parent_map.get(main_place.dcid, [])
 
+  resolved_entity_dcids = set([e.dcid for e in resolved_entities])
+  entities_str = [
+      n for n, d in place_dcids.items() if d in resolved_entity_dcids
+  ]
+  places_str = [
+      n for n, d in place_dcids.items() if d not in resolved_entity_dcids
+  ]
```

Comment:
wonder if its more readable to loop once and set within the loop?

```
for n, d in place_dcids.items():
   if d in resolved_entity_dcids:
      ...
   else:
      ...
```

Reply:
done